Tasks/Common/azurerest-common/azurerestutility.ts

```diff
@@ -348,14 +363,20 @@ async function getOperationStatus(SPN, url: string) {
     var headers = {
         authorization: 'Bearer ' + accessToken
     };
-    httpObj.get('GET', url, headers, (error, response, body) => {
-        if (error) {
-            deferred.reject(error);
-        }
-        else {
-            deferred.resolve({ "response": response, "content": body } );
-        }
+
+    let options: rm.IRequestOptions = {};
+    options.additionalHeaders = headers;
+    let promise: Promise<rm.IRestResponse<any>> = rc.get(url, options);
+    promise.then((response) => {
+        deferred.resolve({
+            "response": response.statusCode,
```

Comment:
Can you make corresponding changes in the monitorSlotSwap method to use 'response' directly instead of 'response.statusCode' and 'response.StatusMessage'